CHANGELOG.md

```diff
@@ -8,12 +8,19 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 and follows a [Backwards Compatibility Policy](https://docs.anza.xyz/backwards-compatibility)
 
 Release channels have their own copy of this changelog:
-* [edge - v3.0](#edge-channel)
-* [beta - v2.3](https://github.com/anza-xyz/agave/blob/v2.3/CHANGELOG.md)
-* [stable - v2.2](https://github.com/anza-xyz/agave/blob/v2.2/CHANGELOG.md)
+* [edge - v3.1](#edge-channel)
+* [beta - v3.0](https://github.com/anza-xyz/agave/blob/v3.0/CHANGELOG.md)
+* [stable - v2.3](https://github.com/anza-xyz/agave/blob/v2.3/CHANGELOG.md)
 
 <a name="edge-channel"></a>
-## 3.0.0 - Unreleased
+## 3.1.0—Unreleased
+### RPC
+#### Breaking
+#### Changes
+### Validator
+#### Breaking
```

Comment:
FWIW, we have added these sections as needed in the past; maybe it is better to have the sections from the start tho so they are utilized properly